src/olympia/blocklist/models.py

```diff
@@ -311,9 +312,11 @@ def process_input_guids(cls, guids, v_min, v_max, load_full_objects=True,
             list(block_qs)
             if load_full_objects else
             [cls.FakeBlock(
-                id_, guid, addon_guid_dict.get(guid), min_version, max_version)
-             for id_, guid, min_version, max_version in block_qs.values_list(
-                'id', 'guid', 'min_version', 'max_version')])
+                id_, guid, addon_guid_dict.get(guid), min_version, max_version,
+                (kinto_id.startswith('*')))
```

Comment:
keyword arguments here as well. (the whole `existing_blocks = (...)` bit is hard to read, I don't think trying to squeeze all that in the same "line" helps...

Reply:
refactored the entire method to separate the cases where we're using fake objects and full django model instances https://github.com/mozilla/addons-server/pull/13683/commits/e8b281a875fd11b96e533c8933fb63b73405aef6